src/main/resources/static/assets/css/style.css

```diff
@@ -21673,7 +21673,6 @@ a.material-icons-shopping_cart:hover {
   display: block;
   position: absolute;
   text-align: left;
-  margin-top: 2.3125rem;
```

Comment:
I did one more change - I added some margin between dropdown and navigation item so that it aligns well when navigation is stick to the page top when scrolling down i.e.:

![image](https://user-images.githubusercontent.com/3949308/27279279-741f6ad0-54e4-11e7-8aca-e62159d9c025.png)

after scroll down:
![image](https://user-images.githubusercontent.com/3949308/27279327-9a4fbd22-54e4-11e7-8c49-565a74a9388b.png)